### DISCUSSION TITLE
where are rx tx pins on esp32s2 mini

### DISCUSSION THREAD

**Discussion by MartinMirwald** [luc-github/ESP3D#904] [F.A.Q] [answered]
I got esp3d working on esp32 s2 mini and can access the web ui but I cant find a pin out diagram of esp32s2 mini showing rx and tx pins, and dont know how to connect it

**Comment by luc-github** [comment]
do you have a picture of your board ?

**Comment by MartinMirwald** [comment]
![16789901646907510742896967059495](https://user-images.githubusercontent.com/45106049/225713082-84f29871-2f15-4a8f-a02b-5ee3c1a42e05.jpg)
![16789901858503625546886811090729](https://user-images.githubusercontent.com/45106049/225713167-11d1e037-ef37-4fc2-a33e-b70d5cdcd721.jpg)

**Reply by luc-github** [reply]
for what I can read here :https://www.wemos.cc/en/latest/s2/s2_mini.html
the rx/tx pins are not exposed, but the Rx1(gpio18)/Tx1(gpio17) are, so I suggest you to use the serial1 pins for the communication

**Reply by MartinMirwald** [reply]
Thanks, i'll try that

**Reply by MartinMirwald** [reply]
Thanks, ist worked, Just Had top Change USE_SERIAL_0 to USE_SERIAL_1 and connected the Pins 18 and 17 Like you said

**Comment by luc-github** [comment, marked as answer]
Great so I mark using Serial1 is the answer
